README.md

```diff
@@ -32,18 +32,48 @@ This code requires Python 3.6 or a higher version. If you haven’t already, dow
 pip install -r requirements.txt
 ```
 
+#### Environment variable
 
+Please set up the following environmet variable for your local computer before running the script.
 
-### Execution
+Runtime Environment Variables:
 
-#### Step 1: Run the translator script (from Work_Zone/translator/source code/Folder)
+| Name      | Value | Description    |
+| :---        |    :----:   |          ---: |
+| contact_name     | Abinash Konersman       |  name of environment variable contact_name containing name of the cotact person  |
+| contact_email   | abinash.konersman@state.co.us       | name of contact email containing email  of contact person |
+| issuing_organization   | CDOT       | name of the oraganization     |
```

Comment:
This should be `organization` rather than `oraganization`.

Reply:
should be fixed